ush/generate_FV3LAM_wflow.sh

```diff
@@ -597,12 +597,15 @@ $settings"
 #
 #-----------------------------------------------------------------------
 #
-print_info_msg "
-Creating symlink in the experiment directory (EXPTDIR) that points to the
-workflow launch script (WFLOW_LAUNCH_SCRIPT_FP):
-  EXPTDIR = \"${EXPTDIR}\"
-  WFLOW_LAUNCH_SCRIPT_FP = \"${WFLOW_LAUNCH_SCRIPT_FP}\""
-ln -fs "${WFLOW_LAUNCH_SCRIPT_FP}" "$EXPTDIR"
+print_info_msg "Generating the workflow launch script
+${EXPTDIR}/${WFLOW_LAUNCH_SCRIPT_FN}"
+
+cat > ${EXPTDIR}/${WFLOW_LAUNCH_SCRIPT_FN} <<EOF
+#!/bin/bash
+module load rocoto
+rocotorun -w "${WFLOW_XML_FN}" -d "${WFLOW_XML_FN%.*}.db"
+EOF
+
```

Comment:
@guoqing-noaa, I understand that a huge log file is generated for a DA run by the launch script. The rocoto xml file for a DA run is incomplete. There are so many tasks unlaunched each cycle. Some useful capabilities of the launch script are not used in a DA run. However, this script is very useful for a non-DA run. So I suggest you to put an if-statement here like
```
if [ "${DO_NON_DA_RUN}" = "TRUE" ]; then
  (current link)
else
  (your new script)
fi

Reply:
@guoqing-noaa, in addition, `module load rocoto` doesn't work on WCOSS2. Please check `modulefiles/wflow_wcoss2.lua`.

Reply:
@guoqing-noaa, I am looking through the launch script. As I mentioned above, it has some useful functions can be used when a rocoto xml file is complete. So my new suggestion is: how about generating a new script with a new name without removing the current link? I'll update the launch script later.

Reply:
@chan-hoo Thanks for the suggestion. It is a good idea to use an alternate name for a simplified version. I will make the change soon.

Reply:
Regarding the heavy launch script, I have alternate thoughts:
1. Users can always run rocotostat to check task status manually. So running rocotostat twice in each crob job is not a must. If the cycledef spans a few months (for example, 4 months in my RTMA run situation), the log file will grow very big in size quickly.
2. var_defns.sh is a huge file with 2991 lines. Sourcing it in each cron job but only using 3 or 4 variables is not that efficient for a cron job.
3. I understand that rocotorun may fail to submit some tasks due to various issues. However, the `FV3LAM_wflow_*.log` file may already provide some useful information for debugging.  If an issue is transient, we may not need to or may not be able to address it. If an issue is persistent, we can always debug based on the current cycle runs.

Just my two cents. Thanks!